crates/core/src/json/client_api.rs

```diff
@@ -78,8 +79,8 @@ pub struct SubscriptionUpdateJson {
 #[derive(Debug, Clone, Serialize)]
 pub struct EventJson {
     pub timestamp: u64,
-    pub status: String,          // committed, failed
-    pub caller_identity: String, // hex identity
+    pub status: String,            // committed, failed
+    pub caller_identity: Identity, // hex identity
```

Comment:
`Identity` is not serialized through hex afaics, so this is essentially breaking. Not sure what consumes this value, though.

Is this change necessary for what this patch is trying to achieve?